packages/zowe-explorer-api/src/profiles/ZoweExplorerApi.ts

```diff
@@ -147,7 +124,7 @@ export namespace ZoweExplorerApi {
          * @param {zowe.IUploadOptions} [options]
          * @returns {Promise<zowe.IZosFilesResponse>}
          */
-        putContent?(inputFilePath: string, ussFilePath: string, options?: zowe.IUploadOptions): Promise<zowe.IZosFilesResponse>;
+        putContent(inputFilePath: string, ussFilePath: string, options?: zowe.IUploadOptions): Promise<zowe.IZosFilesResponse>;
```

Comment:
Hey @JillieBeanSim, I would like to get your thoughts on this comment
https://github.com/zowe/vscode-extension-for-zowe/pull/2082#discussion_r1123669531

It's about possibly keeping the plural form (to be consistent with its `getContents` counterpart) but keep the `options` object the same.

Reply:
I understand everyone's OCD with the names and I'm sure that's why the old one was named with the plural, but since the APIs are consumed by extenders and the one with plural name is the one that is deprecated and to be removed I assumed the other would become the main call.  Some extenders have already moved over to the new call since it has been available for some time now.

Reply:
we could create a v3 story to go over this file and name things appropriately, it will create more work for extenders to adopt v3 though.